packages/zoe/test/swingsetTests/zoe/vat-alice.js

```diff
@@ -49,7 +44,8 @@ const build = async (E, log, zoe, issuers, payments, installations, timer) => {
     const bobInvite = E(publicAPI).makeInvite();
     await E(bobP).doAutomaticRefund(bobInvite);
     const payout = await payoutP;
-    const [moolaPayout, simoleanPayout] = await Promise.all(payout);
+    const moolaPayout = await payout.Contribution1;
+    const simoleanPayout = await payout.Contribution2;
```

Comment:
Don't sequential `await`s turn this into two turns? Wouldn't it work to `await payout` once, and then destructure the result?

Reply:
We need to `await` the individual promises for each payment. So, we could do the equivalent of a promise.all but for objects (I think that's a helper we have somewhere). But, I don't think how it is currently is wrong or hurts us, since it's a test.

Reply:
Just btw, a related issue: For test code only, I've come to like `await`. Tests are normally written to run without the possibility of interference from interleavings. Tests are normally written to read as a narrative "this happens" then perhaps after that settles down, "that happens". In a standalone world where nothing but that one test is running.

This does not reduce the urgency of removing `await` from our own code. This split policy though does create an expository problem: The tests no longer serve as examples, and no longer test as well the way production code should use what is being tested.

None of my comments above suggest a change to the PR (unless there's an `await` in non-test code that I missed). Just btw.